doc/conf.py

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
 # -*- coding: utf-8 -*-
```

Comment:
I think this must be the first line, but it is not really needed now. Lets remove it later.

Reply:
https://github.com/oVirt/vdsm/issues/326